Makefile

```diff
@@ -5,7 +5,8 @@ OTELCOL_BUILDER_VERSION ?= 0.97.0
 OTELCOL_BUILDER_DIR ?= ${HOME}/bin
 OTELCOL_BUILDER ?= ${OTELCOL_BUILDER_DIR}/ocb
 
-DISTRIBUTIONS ?= "otelcol,otelcol-contrib"
+DISTRIBUTIONS ?= "otelcol,otelcol-contrib,otelcol-k8s"
+GEN_CONFIG_DISTRIBUTIONS ?= "otelcol,otelcol-contrib"
```

Comment:
Is this `GEN_CONFIG_DISTRIBUTIONS` needed? How is this used?

Reply:
I believe this is because Tyler disabled generating the goreleaser config from metadata, and instead used a custom one, as described in this [comment](https://github.com/open-telemetry/opentelemetry-collector-releases/pull/507#issuecomment-2025755403).

Reply:
@swiatekm-sumo is correct, but it seems I've got a divergence between this branch and my fork's main branch I was testing on.  This should be used in `make generate-goreleaser`.  I will investigate.

Reply:
Turns out main is broken: https://github.com/open-telemetry/opentelemetry-collector-releases/pull/517

Reply:
Should now be fixed - `GEN_CONFIG_DISTRIBUTIONS` is now correctly used in `make generate-goreleaser`.